src/testHelpers/jest-setup.js

```diff
@@ -18,6 +18,9 @@ global.fetch = fetch;
 global.shallow = shallow;
 global.mount = mount;
 
+process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN =
+  'http://localhost.bbc.com:7080';
```

Comment:
I've made this `localhost` because this (test)[https://github.com/bbc/simorgh/blob/latest/src/server/index.test.jsx#L259] expects `localhost` to be called in the assetOrigin and not `foo`.